src/models/Configuration.php

```diff
@@ -132,6 +133,39 @@ private static function configurationFromRow(
     );
   }
 
+  public static function genFacebookOAuthSettingsExists(): bool {
```

Comment:
FBCTF is mostly following the FB standard of 'gen prefix means returns Awaitable'; google oauth is currently an exception, but shouldn't be.

Reply:
Not a problem.  We will update this.